src/main/webapp/app/overview/course-card.component.html

```diff
@@ -2,101 +2,90 @@
+                                            totalReachableScore: totalReachableScore,
+                                            totalRelativeScore: totalRelativeScore,
+                                        }"
+                                    ></div>
+                                </div>
+                            </div>
+                        } @else {
+                            <div class="fw-bold text-secondary" jhiTranslate="artemisApp.studentDashboard.cardScore"></div>
+                            <div class="fw-bold">
+                                <span jhiTranslate="artemisApp.studentDashboard.noStatistics"></span>
+                            </div>
+                        }
+                        <hr />
+                        <div class="fw-bold text-secondary" jhiTranslate="artemisApp.studentDashboard.cardExerciseLabel"></div>
+                        @if (nextRelevantExercise) {
+                            <a class="fw-bold btn-exercise exercise-title exercise-guided-tour" [routerLink]="['/courses', course.id!, 'exercises', nextRelevantExercise.id!]">
+                                {{ nextRelevantExercise.title }}
+                            </a>
+                        } @else {
+                            <div class="fw-bold mb-3 exercise-guided-tour">
+                                <span jhiTranslate="artemisApp.studentDashboard.cardNoExerciseLabel"></span>
                             </div>
                         }
                     </div>
+                    @if (exerciseCount > 0 && (totalReachableScore > 0 || totalAbsoluteScore > 0)) {
+                        <div class="align-self-center chart-container col-5">
+                            <ngx-charts-pie-chart
+                                id="score-chart"
+                                class="chart-level"
+                                style="position: absolute"
+                                [view]="[160, 160]"
+                                [results]="ngxDoughnutData"
+                                [arcWidth]="0.3"
+                                [scheme]="ngxColor"
+                                [doughnut]="true"
+                                [animations]="false"
+                                (select)="onSelect()"
+                                (click)="onSelect()"
+                            >
+                                <ng-template #tooltipTemplate let-model="model">
+                                    <span jhiTranslate="artemisApp.courseOverview.statistics.{{ model.name }}"></span>
+                                    <span>: {{ model.value }}</span>
+                                </ng-template>
+                            </ngx-charts-pie-chart>
+                        </div>
+                    }
                 </div>
             </div>
+            @if (course.isAtLeastTutor || course.isAtLeastEditor || course.isAtLeastInstructor) {
```

Comment:
Aren't instructors and editors already "at least tutors"?

```suggestion
            @if (course.isAtLeastTutor) {
```